src/light.cpp

```diff
@@ -29,33 +29,58 @@ static u8 light_LUT[LIGHT_SUN + 1];
 // The const ref to light_LUT is what is actually used in the code
 const u8 *light_decode_table = light_LUT;
 
+struct LightingParams {
+	float a, b, c; // polynomial coefficients
+	float boost, center, sigma; // normal boost parameters
+	float gamma;
+};
+
+static LightingParams params;
+
+float decode_light_f(float x)
+{
+	// x is equal to 1.0f most of time, so paramat asked me to add this. -- numzero
```

Comment:
No need for `, so paramat asked me to add this. -- numzero`

Reply:
Sorry but I don’t want to be blamed for that line.